package/doc/sphinx/source/conf.py

```diff
@@ -46,16 +46,8 @@ def __getattr__(cls, name):
 # documentation root, use os.path.abspath to make it absolute, like shown
 # here.
 
-# Little trick to make sure sphinx use the dev build if present
```

Comment:
yes yes and double yes. Without modifying the path sphinx won't find the changes made on the current branch but what ever is installed globally. The old one is not needed anymore IMHO since it only works for python2.x, in Py3 they changed the build dir names. But yeah we still need the new one. It did cost me quite some cursing and time to figure out that path was wrong.